extensions/amp-fit-text/0.1/test/test-amp-fit-text.js

```diff
@@ -1,10 +1,6 @@
 import {createElementWithAttributes} from '#core/dom';
-import {
-  AmpFitText,
-  buildDom,
-  calculateFontSize_,
-  updateOverflow_,
-} from '../amp-fit-text';
+import {AmpFitText, calculateFontSize_, updateOverflow_} from '../amp-fit-text';
+import {buildDom} from '../build-dom';
```

Comment:
One suggestion: the extension file could always `export {buildDom} from './build-dom';` and much of this wouldn't require updating. I don't feel strongly about this either way

Reply:
I actually wrote it this way at one point to make the diff smaller. Decided against it because we don't actually reexport often in the codebase and we don't have anything stopping folks from importing directly from the `build-dom` file. 🤷